configuration/concepts/lab_tests.csv

```diff
@@ -90,40 +90,40 @@ e901a01d-8297-43f4-b39d-05e5ce09b708,,,Blackish,Noirâtre,Blackish,Noirâtre,,,M
+3cf0a904-b1ad-4c35-b2d3-20634e2c4820,,CIEL:679,Red Blood Cell Count,Taux d’hématies,Red Blood Cell Count,Taux d’hématies,,,LabTest,Numeric,,,,Yes,/mm3,3500000,5900000,,,,
+3fb9fc5a-3a35-4298-9a92-c24853adf4f9,,CIEL:1015,Hematocrit,Hematocrite,Hematocrit,Hematocrite,,,LabTest,Numeric,,,,Yes,%,37,52,,,,
+becd4484-4379-4069-8448-d7211f3654c4,,CIEL:21,Hemoglobin,Hemoglobine,Hemoglobin,Hemoglobine,,,LabTest,Numeric,,,,Yes,g/dL,11,18,,,,
+26ffad22-33ed-433f-8430-99fcafcb4baa,,CIEL:1022,Neutrophils,Neutrophiles,Neutrophils,Neutrophiles,,,LabTest,Numeric,,,,Yes,%,50,65,,,,
+f8bd08fd-0243-4ca6-8257-c0ecbddde480,,CIEL:1024,Eosinophils,Eosinophiles,Eosinophils,Eosinophiles,,,LabTest,Numeric,,,,Yes,%,0,4,,,,
+1a951978-61d9-4ef6-9481-d2456f3b4b0c,,CIEL:1025,Basophils,Basophiles,Basophils,Basophiles,,,LabTest,Numeric,,,,Yes,%,0,1,,,,
+86cea01d-37ef-4275-9f28-05997a759c02,,CIEL:1021,Lymphocytes,Lymphocites,Lymphocytes,Lymphocites,,,LabTest,Numeric,,,,Yes,%,10,40,,,,
+4fb44f2b-cb12-488d-b8de-3f4a7917e45f,,CIEL:1023,Monocytes,Monocytes,Monocytes,Monocytes,,,LabTest,Numeric,,,,Yes,%,2,4,,,,
+3b878b87-844e-47de-97b5-50d9fbb3f4d3,,CIEL:1327,Reticulocytes,Reticulocytes,Reticulocytes,Reticulocytes,,,LabTest,Numeric,,,,Yes,%,0,6,,,,
+5c64ff49-2821-476c-af50-8dab7275eeed,,CIEL:851,MCV,MCV,MCV,MCV,,,LabTest,Numeric,,,,Yes,Fl,85,95,,,,
+ec4c96b4-1da4-41b6-978b-4d71d2821153,,CIEL:1018,MCH,MCH,MCH,MCH,,,LabTest,Numeric,,,,Yes,Pg,32,36,,,,
+137e9ba3-85ad-4fe3-bcbc-f05bc4613d34,,CIEL:1017,MCHC,MCHC,MCHC,MCHC,,,LabTest,Numeric,,,,Yes,mg/dl,29,32,,,,
+28241008-8e07-47ec-8692-3c276e6fae4a,,CIEL:729,Platelets,Plaquettes,Platelets,Plaquettes,,,LabTest,Numeric,,,,Yes,/mm3,150000,400000,,,,
 e1a77dc7-c7d6-41d2-9c4e-93acc1093bcf,,CIEL:1019,Complete Blood Count,Hémogramme complet,Complete Blood Count,Hémogramme complet,,,LabSet,N/A,,,White Blood Cell Count;Red Blood Cell Count;Hematocrit;Hemoglobin;Neutrophils;Eosinophils;Basophils;Lymphocytes;Monocytes;Reticulocytes;MCV;MCH;MCHC;Platelets,,,,,,,,
-7036964e-e177-465e-8af8-b1b761f290c5,,CIEL:855,Sedimentation Rate,Vitesse de sédimentation,Sedimentation Rate,Vitesse de sédimentation,,,LabTest,Numeric,,,,,,,,,,,
-15ecef64-f110-4c05-b04c-566610403139,,, 1/20, 1/20, 1/20, 1/20,,,Misc,N/A,,,,,,,,,,,
-3e593c05-35c7-453b-bd07-edc5c786043e,,, 1/40, 1/40, 1/40, 1/40,,,Misc,N/A,,,,,,,,,,,
-b0b79ce3-14fc-4691-960e-cd8b2486616c,,, 1/80, 1/80, 1/80, 1/80,,,Misc,N/A,,,,,,,,,,,
-e6133b5c-f0eb-4c2b-bc3b-408decf7ed21,,, 1/160, 1/160, 1/160, 1/160,,,Misc,N/A,,,,,,,,,,,
-b814124d-2dbd-4777-95c8-f26e936c03ea,,, 1/320, 1/320, 1/320, 1/320,,,Misc,N/A,,,,,,,,,,,
+7036964e-e177-465e-8af8-b1b761f290c5,,CIEL:855,Sedimentation Rate,Vitesse de sédimentation,Sedimentation Rate,Vitesse de sédimentation,,,LabTest,Numeric,,,,Yes,mm/h,0,20,,,,
+15ecef64-f110-4c05-b04c-566610403139,,," 1/20"," 1/20"," 1/20"," 1/20",,,Misc,N/A,,,,,,,,,,,

```

Comment:
Looks like there is a voluntary _space_ before the FSN. Why is that?
I would think this could create some unexpected errors/misbehaviors.